apps/mobile/src/app/(home)/hardware-wallets.tsx

```diff
@@ -0,0 +1,106 @@
+import { useRef, useState } from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { NotifyUserSheet, OptionData } from '@/components/sheets/notify-user-sheet.layout';
+import { Trans, t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+
+import {
+  Box,
+  Cell,
+  LogoHardwareBitkey,
+  LogoHardwareFoundation,
+  LogoHardwareLedger,
+  LogoHardwareOnekey,
+  LogoHardwareRyder,
+  LogoHardwareTrezor,
+  QuestionCircleIcon,
+  SheetRef,
+  Text,
+  Theme,
+  TouchableOpacity,
+} from '@leather.io/ui/native';
+
+function getUnavailableFeatures() {
+  return {
+    bitgo: { title: t`Bitkey`, Icon: LogoHardwareBitkey },
+    capsule: { title: t`Ledger`, Icon: LogoHardwareLedger },
+    copper: { title: t`OneKey`, Icon: LogoHardwareOnekey },
+    fireblocks: { title: t`Passport`, Icon: LogoHardwareFoundation },
+    foredefi: { title: t`Ryder`, Icon: LogoHardwareRyder },
+    portal: { title: t`Trezor`, Icon: LogoHardwareTrezor },
+  };
+}
+
+export default function HardwareWalletsScreen() {
```

Comment:
Can we make a layout component here that separates the logic from the styles 🙏🏼

Reply:
Agree here, these components are quite hard to read.